packages/browser/src/plugins/ajs-destination/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Integrations } from '@segment/analytics-core'
+import { LegacyIntegrationConfiguration } from '../..'
+
+export const isInstallableIntegration = (
+  name: string,
+  integrationSettings: LegacyIntegrationConfiguration
+) => {
+  const { type, bundlingStatus, versionSettings } = integrationSettings
+  // We use `!== 'unbundled'` (versus `=== 'bundled'`) to be inclusive of
+  // destinations without a defined value for `bundlingStatus`
+  const deviceMode =
+    bundlingStatus !== 'unbundled' &&
+    (type === 'browser' || versionSettings?.componentTypes?.includes('browser'))
+
+  // checking for iterable is a quick fix we need in place to prevent
+  // errors showing Iterable as a failed destiantion. Ideally, we should
+  // fix the Iterable metadata instead, but that's a longer process.
+  return (deviceMode || name === 'Segment.io') && name !== 'Iterable'
```

Comment:
`Segment.io` is its own plugin, so shouldn't be included as one of the ajs-destination plugins. 

The previous code had the follow check that filters out the `Segment.io` plugin:
```
if (name.startsWith('Segment')) {
  return
}
```

Suspect we can change this to `return (deviceMode && name !== 'Iterable')` and add a check to exclude `Segment` at the top of the function.

Reply:
I suggest we do this in a separate pull request. Probably not a safe bet to add too many behavourial changes at once, especially when a behaviour has been there for a long time.

Reply:
What you have is a behavioral change though. The code is now saying `Segment.io` is an installable integration. Previously, no `Segment` plugin would ever make it to this line because we did an early `startsWith('Segment')` check.

Reply:
I see, just checked. Since there was an early return does this mean that [this ](https://github.com/segmentio/analytics-next/blob/222d4ecd4db7376839aa816b4b8923cae4f1487c/packages/browser/src/plugins/ajs-destination/index.ts#L351)has been redundant all along?

Reply:
Yeah, the Segment.io check was always redundant.